src/actions/indexing/indexable-post-indexation-action.php

```diff
@@ -144,7 +144,7 @@ protected function get_count_query() {
 			AND P.ID not in (
 				SELECT I.object_id from $indexable_table as I
 				WHERE I.object_type = 'post'
-				AND I.version = %d )",
+				AND I.version >= %d )",
```

Comment:
This seems quite dangerous. This means that indexables might not get updated when users revert to an older version of yoast SEO in the future. For example with the relative permalink change coming up, if people experience problems and have to revert we also want the permalink to be recalculated to the older version.